odoo/tests/common.py

```diff
@@ -899,7 +910,8 @@ def start_screencast(self):
             os.makedirs(self.screencasts_dir, exist_ok=True)
             self.screencasts_frames_dir = os.path.join(self.screencasts_dir, 'frames')
             os.makedirs(self.screencasts_frames_dir, exist_ok=True)
-        self._websocket_send('Page.startScreencast', params={'maxWidth': 1024, 'maxHeight': 576})
+        width, height = (int(size) for size in self.window_size.split('x'))
+        self._websocket_send('Page.startScreencast', params={'maxWidth': width, 'maxHeight': height})
```

Comment:
I'm not sure params are needed anymore, I guess that the default max is the window size. 
```suggestion
        self._websocket_send('Page.startScreencast')
```

Reply:
You cannot be sure with the [experimental devtools protocol ](https://chromedevtools.github.io/devtools-protocol/tot/Page/#method-startScreencast).
But let's try